src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -96,7 +96,11 @@ public static ControlFlowGraph Create(IOperation body, ControlFlowGraph? parent
                     body.Kind == OperationKind.ConstructorBody ||
                     body.Kind == OperationKind.FieldInitializer ||
                     body.Kind == OperationKind.PropertyInitializer ||
-                    body.Kind == OperationKind.ParameterInitializer,
+                    body.Kind == OperationKind.ParameterInitializer ||
+                    // `AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer` is passing an Attribute operation.
```

Comment:
I'm not sure why we would remove this line? It seems like you should be able to get a CFG for an attribute, even if it will be boring in most cases.